lib/models/auth.dart

```diff
@@ -6,23 +6,26 @@ import 'package:firebase_auth/firebase_auth.dart';
  Last thing i did was move the file into the Models folder, figured it would make more sense
 */
 
-
+/// The authentication Class, used for logging in users.
```

Comment:
Just so you know, any comments that have a triple slash will appear when moused over in an IDE, useful for when working as a team.

Reply:
@Neechie